bindgen/templates/RecordTemplate.cs

```diff
@@ -6,6 +6,10 @@
 {%- let (ordered_fields, is_reordered) = rec.fields()|order_fields %}
 
 {%- call cs::docstring(rec, 0) %}
+{%- for field in ordered_fields %}
+{%- call cs::param_docstring(field.name(), field, 0) %}
```

Comment:
This doesn't work. I get a build error saying that `cs::param_docstring` doesn't exist. How do I expose the function I added?

Reply:
I think you invoke it from within `{{ }}` block, e.g. `{{ param_docstring(field.name(), field, 0) }}`

Reply:
But I think for this special case it's possible to avoid another Rust function.
```suggestion
{%- match field.docstring() %}
{%- when Some with(docstring) %}
/// <param name="{{ field.name() }}">
{%- for line in docstring.lines() %}
/// {{ textwrap::dedent(line) }}
{%- endfor %}
/// </param>
{%- else %}
{%- endmatch %}
```